tools/ui

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$1" == "run" ]; then
```

Comment:
Would be helpful if you add comment to explain what this is and maybe also usage examples.